src/submission/models.py

```diff
@@ -1226,6 +1226,9 @@ def non_correspondence_authors(self):
         else:
             return self.author_accounts
 
+    def is_submitted(self):
+        return self.stage != STAGE_UNSUBMITTED
```

Comment:
Should this return `True` for articles that are submission complete? how about imported articles? I'd avoid establishing this API until these are resolved, and since this is esentially a single statement, I doubt the usefulness of its use is worth more than the potential misuse it could introduce